doc/getting-started.md

```diff
@@ -0,0 +1,313 @@
++++
+title = "Getting started"
+docpage = true
+[menu.docs]
+  parent = "getting-started"
++++
+
+Conduit has two basic components: a *data plane* comprised of lightweight
+proxies, which are deployed as sidecar containers alongside your service code,
+and a *control plane* of processes that coordinate and manage these proxies.
+Humans interact with the service mesh via a command-line interface (CLI) or
+a web app that you use to control the cluster.
+
+In this guide, we’ll walk you through how to deploy Conduit on your Kubernetes
+cluster, and how to set up a sample gRPC application.
+
+Afterwards, check out the [Using Conduit to debug a service](/debugging-an-app) page,
+where  we'll walk you through how to use Conduit to investigate poorly performing services.
+
+> Note that Conduit v{{% latestversion %}} is an alpha release. Conduit will
+automatically work for most protocols. However, applications that use
+WebSockets, HTTP tunneling/proxying, or protocols such as MySQL and SMTP, will
+require some additional configuration. See [Adding your service to the
+mesh](/adding-your-service) for details.
+
+____
+
+##### STEP ONE
+## Set up 🌟
+First, you'll need a Kubernetes cluster running 1.8 or later, and a functioning
+`kubectl` command on your local machine.
+
+To run Kubernetes on your local machine, we suggest
+<a href="https://kubernetes.io/docs/tasks/tools/install-minikube/" target="_blank">Minikube</a>
+ --- running version 0.24.1 or later.
+
+### When ready, make sure you're running the latest version of Kubernetes with:
+#### `kubectl version --short`
+
+### Which should display:
+```
+Client Version: v1.8.3
+Server Version: v1.8.0
+```
+Confirm that both `Client Version` and `Server Version` are v1.8.0 or greater.
+If not, or if `kubectl` displays an error message, your Kubernetes cluster may
+not exist or may not be set up correctly.
+
+___
+
+##### STEP TWO
+## Install the CLI 💻
+If this is your first time running
+Conduit, you’ll need to download the command-line interface (CLI) onto your
+local machine. You’ll then use this CLI to install Conduit on a Kubernetes
+cluster.
+
+### To install the CLI, run:
+#### `curl https://run.conduit.io/install | sh`
+
+### Which should display:
+```
+Downloading conduit-{{% latestversion %}}-macos...
+Conduit was successfully installed 🎉
+Copy $HOME/.conduit/bin/conduit into your PATH.  Then run
+    conduit install | kubectl apply -f -
+to deploy Conduit to Kubernetes.  Once deployed, run
+    conduit dashboard
+to view the Conduit UI.
+Visit conduit.io for more information.
+```
+
+>Alternatively, you can download the CLI directly via the
+[Conduit releases page](https://github.com/runconduit/conduit/releases/v{{% latestversion %}}).
```

Comment:
Similar to the comment above, this `latestversion` shortcode also looks pretty funky on GitHub:

![image](https://user-images.githubusercontent.com/9226/36709817-c3ce64da-1b2e-11e8-8283-1b130d7af88f.png)

Maybe we can switch all occurrences to some sort of placeholder string (`LATESTVERSION`?), and then convert the string into the template shortcode when importing the doc into the site repo.